views/js/controller/runtime/testRunner.js

```diff
@@ -720,13 +720,15 @@ function (
                                 timerIndex = i;
 
                                 if (self.testContext.timerWarning && self.testContext.timerWarning[cst.qtiClassName]) {
-                                    cst.warnings = self.testContext.timerWarning[cst.qtiClassName];
-                                    _(cst.warnings).forEach(function (value, key) {
-                                        cst.warnings[key] = {
-                                            type: value,
-                                            showed: cst.seconds <= key,
-                                            point: parseInt(key, 10)
-                                        };
+                                    cst.warnings = {};
+                                    _(self.testContext.timerWarning[cst.qtiClassName]).forEach(function (value, key) {
+                                        if (_.indexOf(['info', 'warning', 'danger'], value) != -1) {
```

Comment:
Use either `(_.indexOf(['info', 'warning', 'danger'], value) !== -1)` (to use consistenl rules)  or `(_.contains(['info', 'warning', 'danger'], value))` if you don't mind of the index